marketplace/commercelayer-sku/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "commercelayer-sku",
+  "version": "0.1.0",
+  "private": true,
+  "devDependencies": {
+    "@babel/core": "7.3.4",
+    "@babel/plugin-proposal-class-properties": "7.3.4",
+    "@babel/plugin-transform-runtime": "7.3.4",
+    "@babel/preset-env": "7.3.4",
+    "@babel/preset-react": "7.0.0",
+    "@contentful/contentful-extension-scripts": "0.11.0",
+    "cssnano": "4.1.10",
+    "contentful-cli": "0.28.0"
+  },
+  "dependencies": {
+    "@contentful/forma-36-fcss": "^0.0.20",
+    "@contentful/forma-36-react-components": "^3.11.2",
+    "@contentful/forma-36-tokens": "^0.3.0",
+    "contentful-ui-extensions-sdk": "3.9.0",
+    "prop-types": "^15.7.2",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "vue": "^2.6.10"
+  },
+  "scripts": {
+    "start": "contentful-extension-scripts start",
+    "build": "contentful-extension-scripts build",
+    "deploy": "npm run build && contentful extension update --force",
```

Comment:
... and add `test` and `test:coverage` command here, similar to what other marketplace extensions have

Reply:
Done